packages/mold-bin/mold-bin.pacscript

```diff
@@ -0,0 +1,23 @@
+#  __      __________   ______
+# /  \    /  \_____  \ /  __  \
+# \   \/\/   //  ____/ >      <
+#  \        //       \/   --   \
+#   \__/\  / \_______ \______  /
+#        \/          \/      \/
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="mold-bin"
+gives="mold"
+version="1.11.0"
+arch=("amd64")
+url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
```

Comment:
You could add a `case "${CARCH}"` part to make this pacscript available on arm and arm64 as well.

Reply:
No